docs/proposals/20200506-conditions.md

```diff
@@ -471,6 +473,7 @@ enhance the condition utilities to handle those situations in a generalized way.
 - Risk: This proposal aims to be consistent with the target state of conditions in Kubernetes, but this
   is still under definition (see [KEP](https://github.com/kubernetes/enhancements/pull/1624)).
   - Mitigation: Periodically re-evaluate this proposal vs the Kubernetes KEP. 
+    -  2024-05-03: Change allowing conditions with negative polarity goes into this direction
```

Comment:
```suggestion
    -  2024-05-03: Change to allow conditions without positive polarity goes into this direction
```
(~ means the same but this framing would also include conditions without polarity. I think Paused is basically neiter positive nor negative)

(same in Implementation History)